.travis.yml

```diff
@@ -22,7 +22,7 @@ install:
   - go get $LIBP2P_DAEMON_REPO
   - cd $GOPATH/src/$LIBP2P_DAEMON_REPO
   - make bin
-  # install tox
+  - pip install --upgrade pip
```

Comment:
This is because we need `pip >= 18` to support PEP 508, allowing us to conveniently use dependencies which are not in PyPI.